Client/YooChooseNotifier.php

```diff
@@ -433,16 +426,15 @@ protected function configureOptions(OptionsResolver $resolver)
     }
 
     /**
-     * Returns the YooChoose notification endpoint.
+     * Log message if logger is available.
      *
-     * @return string
+     * @param string $message
+     * @param string $level
      */
-    private function getNotificationEndpoint()
+    private function log($message, $level = 'info')
     {
-        return sprintf(
-            '%s/api/v4/publisher/ez/%s/notifications',
-            rtrim($this->options['api-endpoint'], '/'),
-            $this->options['customer-id']
-        );
+        if (isset($this->logger) && method_exists($this->logger, $level)) {
```

Comment:
You don't need this method if you initialize `$this->logger` with `\Psr\Log\NullLogger` in the constructor.